.github/workflows/build-test.yml

```diff
@@ -47,9 +47,16 @@ env:
 
 jobs:
 
-  # Build job - basic smoke test
-  check-msrv:
+  # ci-job-check & ci-job-features
+  # Running at MSRV (1.61)
+  msrv:
```

Comment:
nit: Can you provide a `ci-job-msrv` task that just prints out "please run `ci-job-features` or  `ci-job-check` with +<msrv>"

That way we retain the property that `ci-job-foo` is helpful when CI task `foo` fails.

Reply:
Added a job that actually does it.